cpp/src/io/comp/gpuinflate.cu

```diff
@@ -1024,7 +1024,7 @@ __device__ int parse_gzip_header(uint8_t const* src, size_t src_size)
  * @param parse_hdr If nonzero, indicates that the compressed bitstream includes a GZIP header
  */
 template <int block_size>
-__global__ void __launch_bounds__(block_size)
+CUDF_KERNEL void __launch_bounds__(block_size)
```

Comment:
Should we take this opportunity to normalize the order of launch bounds, `CUDF_KERNEL`, and the return type across all kernels in libcudf? Some put launch bounds first, others put it last.

Reply:
Happy to make everything consistent as part of the PR, and we can always discuss offline/follow up what style we want. I don't want to hold up the entire PR over a style issue though

Reply:
We can defer on this. It's easy enough to change later. Just wanted to raise that question in case you had a clear preference. I don't know which one I prefer. Maybe `CUDF_KERNEL __launch_bounds__(...) void`, but that doesn't align with any of the kernels that I saw.

Reply:
You could further future proof it (and potentially enable wider compatibility by making a version that takes the bounds as parameters

```
CUDF_KERNEL_WITH_LAUNCH_BOUNDS(...) void foo(...)
```

Or even make all kernels use the same macro with varargs.

```
CUDF_KERNEL(...) void foo(...)
```